test/FilesystemUtils.cpp

```diff
@@ -81,5 +81,6 @@ bool solidity::test::createSymlinkIfSupportedByFilesystem(
 		BOOST_THROW_EXCEPTION(runtime_error(
 			"Failed to create a symbolic link: \"" + _linkName.string() + "\""
 			" -> " + _targetPath.string() + "\"."
+			" " + symlinkCreationError.message() + "."
```

Comment:
When I run tests on my Windows system as a regular user, link creation fails. It works if I run tests as an administrator.

Showing "_A required privilege is not held by the client._" might help to figure out why tests are failing.

Reply:
Interesting. Looks like symlinks on Windows are admin-only feature. And only in [Win 10 MS added some "developer mode"](https://www.atmosera.com/news/non-admin-users-can-now-create-symlinks-windows-10/) that lets you create them as a regular user.

This change looks good but I think we should also mention in the "contributing" docs that our test suite requires privileges for link creation.

Reply:
See 07ed09a255c2d96a853d6cd90cda0ac08d5e1500.